tests/migration/Core/V6_6/Migration1701776000SetSystemDefaultForDefaultMailFooterTest.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopware\Tests\Core\Migration\V6_6;
+
+use PHPUnit\Framework\TestCase;
+use Shopware\Core\Migration\V6_6\Migration1701776000SetSystemDefaultForDefaultMailFooter;
+use Shopware\Tests\Migration\MigrationTestTrait;
+
+/**
+ * @internal
+ *
+ * @covers \Shopware\Core\Migration\V6_6\Migration1701776000SetSystemDefaultForDefaultMailFooter
+ */
+class Migration1701776000SetSystemDefaultForDefaultMailFooterTest extends TestCase
+{
+    use MigrationTestTrait;
+
+    public function testCreationTimestamp(): void
```

Comment:
please also add a proper test case for your migration, 

E.g. adjust the german templates, run your migration, check that the german templates are still system_default: false, but the english ones are set as system default